mlflow/openai/__init__.py

```diff
@@ -558,12 +573,47 @@ def predict(
             }
             for messages in messages_list
         ]
+        results = process_api_requests(requests)
+        return [r["choices"][0]["message"]["content"] for r in results]
+
+    def _predict_embeddings(self, data):
+        import openai
```

Comment:
Can we move `import openai` to the top level?

Reply:
We can't because it's an optional depedency